src/booleans.js

```diff
@@ -1,62 +1,98 @@
-function negate(a) {
-  // your code here
-};
+const negate = (a) => !a;
 
-function both(a, b) {
-  // your code here
-};
+const both = (a, b) => {
+  if (!a || !b) {
+    return false;
+  }
+  return true;
+}
 
-function either(a, b) {
-  // your code here
-};
+const either = (a, b) => {
+  if (a || b) {
+    return true;
+  }
+  return false;
+}
+const none = (a, b) => {
+  if (a || b) {
+    return false;
+  }
+  return true;
+}
 
-function none(a, b) {
-  // your code here
-};
+const one = (a, b) => {
+  if ((a && !b) || (b && !a)) {
+    return true;
+  }
+  return false;
+}
 
-function one(a, b) {
-  // your code here
-};
+const truthiness = (a) => {
+  if(a){
+    return true;
+  }
+    return false;
+  }
 
-function truthiness(a) {
-  // your code here
-};
+const isEqual = (a, b) => {
+  if (a === b){
+    return true;
+  }
```

Comment:
You can just return `a === b` here as the comparison logic evaluates to a boolean already. :)